src/shared/bg_pmove.cpp

```diff
@@ -4249,22 +4249,30 @@ static void PM_Animate()
 	}
 
 	bool doit = false;
+	bool gesture = usercmdButtonPressed( pm->cmd.buttons, BTN_GESTURE );
+	bool rally  = usercmdButtonPressed( pm->cmd.buttons, BTN_RALLY );
 	if ( IsSegmentedModel( pm->ps ) )
 	{
-		if ( usercmdButtonPressed( pm->cmd.buttons, BTN_GESTURE ) )
+		if ( gesture )
 		{
 			int wpAnim = TORSO_GESTURE_BLASTER + ( pm->ps->weapon - WP_BLASTER );
 			//and now I know why build stuff must be last in the weapon list...
 			PM_StartTorsoAnim( wpAnim > WP_LUCIFER_CANNON ?  TORSO_GESTURE_CKIT : wpAnim );
 			doit = true;
 		}
-		else if( usercmdButtonPressed( pm->cmd.buttons, BTN_RALLY ) )
```

Comment:
The else if makes more sense. You can't play two torso animations at once.

Reply:
Agree, but now the code does the same as old, "just in case" again (see my note above it: I think all mentions to rallly could be removed anyway).

Reply:
It's obviously useless to do both at once, so why revert to the old behavior? Even the change to not produce falling sounds for dead players at a certain distance or whatever seemed far more bold than this one :stuck_out_tongue:

Reply:
"so why revert to the old behavior?" because the intent of the commit was *not* to change behaviors but only to make code more readable.
Otherwise, I would have boldly removed every mentions of BTN_RALLY since AFAIK it does not really exists.

Reply:
OTOH since this thing is basically just a no-OP I guess I can keep the else anyway